repose-aggregator/src/docs/asciidoc/recipes/index.adoc

```diff
@@ -26,17 +26,18 @@ Covers the typical Repose setup -- a Valve installation (i.e., standalone applic
 == <<war-installation.adoc#,WAR Installation>>
 Covers setting up Repose inside of a user-provided servlet container.
 
-== <<quick-setup.adoc#,Quick Setup>>
-*IN PROGRESS*
-If you want to stand up a Repose instance quickly, visit the Quick Setup page.
+== <<quick-start.adoc#,Quick Start>>
+If you want to stand up a Repose instance quickly, visit the Quick Start.
```

Comment:
Hm, looking at this again, the following seems a little redundant:
> visit the Quick Start.

This is just the index page, and the section header is already a link with the same works, so I'm not sure that bit is necessary. Just the description should suffice. I might suggest something like:
> Stand up a Repose instance quickly!
or
> Learn how to stand up Repose as quickly as possible!

Reply:
This comment goes for the other sections with lines reading:
`See ...`

Reply:
Updated.